src/page/order-result.page.ts

```diff
@@ -4,7 +4,7 @@ export class OrderResultPage {
   private orderResultAlert: ElementFinder;
 
   constructor() {
```

Comment:
is this page a real page object? or should be part of another one?

Reply:
You made me realize this page object is part from Payment step. However, I thought it was not by reloaded page after confirm order.